site/content/docs/main/backup-hooks.md

```diff
@@ -86,6 +86,23 @@ To use multiple commands, wrap your target command in a shell and separate them
     pre.hook.backup.velero.io/command='["/bin/bash", "-c", "echo hello > hello.txt && echo goodbye > goodbye.txt"]'
 ```
 
+## Using Environment Variables in pre and post hooks
+
+You are able to use environment variables from your pods in your pre and post hook commands by including a shell command before using the environment variable. For example, MYSQL_ROOT_PASSWORD is an environment variable defined in pod called "mysql". To use MYSQL_ROOT_PASSWORD in your pre-hook, you'd include a shell, like `/bin/sh`, before calling your environment variable:
+
+```
+pre:
+- exec:
+    container: mysql
+    command:
+      - /bin/sh
+      - -c
+      - mysql --password=$MYSQL_ROOT_PASSWORD -e "FLUSH TABLES WITH READ LOCK"
+    onError: Fail
+```
+
+Your pod must have integrated a shell to use environment variables.
```

Comment:
May also need to mention different container integrated with different shell, so need to specify the correct shell for container used.

Reply:
i updated the wording a little bit. can you take a look and let me know if this makes sense?

Reply:
Thanks. Looks good to me.